### DISCUSSION TITLE
Installation Failed [Exception]

### DISCUSSION THREAD

**Discussion by AlohaJeff** [breezefront/community#17] [1. Help]
We're running into an error using the installation instructions found here:

https://breezefront.com/installation

Error = Installation failed, reverting ./composer.json to its original content.

[Exception]
Higher matching version 1.2.3 of swissup/theme-frontend-breeze-blank was found in public repository packagist.org than 1.2.2 in private https://repo.magento.com. Public package might've been taken over by a malicious entity, please investigate and update package requirement to match the version from the private repository.

We can overcome this error a number of ways but it's worth asking... is there a better solution that ensures we're accessing the "newest" version of Breeze?

Many mahalos!

**Comment by vovayatsyuk** [comment]
I can suggest disabling Magento's audit plugin in your composer.json:

```
"config": {
    "allow-plugins": {
      ...
      "magento/composer-dependency-version-audit-plugin": false
    }
}
```

Another way to fix this issue is to add `swissup` to the allowed list: https://github.com/magento/composer-dependency-version-audit-plugin/blob/main/src/Plugin.php#L56

**Comment by dimdimich** [comment]
We just checked breezefront/theme-frontend-breeze-blank package at repo.magento.com and it looks that there was issue with it's submission. We just resubmitted it. Please try to install it again in next 15-20 minutes.
